.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+### Type of PR
+
+* [ ] Bugfix
+* [ ] New Feature
+* [ ] Breaking change
```

Comment:
I think "Breaking Change" should be on a separate Line/Section and the third point should be "something else" or "other".

Reply:
Regarding the addition of "else": full ack :+1: 

Regarding the "Breaking change" box: how would you propose to name this section?

I certainly get your point, I wasn't sure about this either, but in the end I thought that nobody forbids you to check multiple boxes (these are intentionally check boxes, not radio boxes). But as I said, I'm not sure about this and happy to discuss :)